Core/src/Material/MaterialProperties.cpp

```diff
@@ -17,20 +17,12 @@ static constexpr auto eps = 2 * std::numeric_limits<float>::epsilon();
 Acts::MaterialProperties::MaterialProperties(float thickness)
     : m_thickness(thickness) {}
 
-Acts::MaterialProperties::MaterialProperties(float X0, float L0, float Ar,
-                                             float Z, float rho,
-                                             float thickness)
-    : m_material(X0, L0, Ar, Z, rho),
-      m_thickness(thickness),
-      m_thicknessInX0((X0 > eps) ? (thickness / X0) : 0),
-      m_thicknessInL0((L0 > eps) ? (thickness / L0) : 0) {}
-
 Acts::MaterialProperties::MaterialProperties(const Material& material,
                                              float thickness)
     : m_material(material),
       m_thickness(thickness),
-      m_thicknessInX0((material.X0() > eps) ? (thickness / material.X0()) : 0),
-      m_thicknessInL0((material.L0() > eps) ? (thickness / material.L0()) : 0) {
+      m_thicknessInX0((eps < material.X0()) ? (thickness / material.X0()) : 0),
+      m_thicknessInL0((eps < material.L0()) ? (thickness / material.L0()) : 0) {
```

Comment:
Really minor point but what is the reasoning for changing the ordering here ? In my head the norm for writing comparison is "variable <, > or == constant".

Reply:
I always try to use `<`-only for order comparison: this orders the components the same way you would order them along a number axis. I find this particularly convenient e.g. for limit checks on both side
```
if ((0 < x) and (x < 1)) { ... }
```
With this notation it is also visually clear that `x` is within the lower and upper limits.

Of course, this is a personal preference.

Reply:
I was mostly just curious, that is an interesting way to look at this. Thanks for the answer !